cmd/fuego/templates/controller/controller.go

```diff
@@ -1,4 +1,4 @@
-package controller
+package controllers
```

Comment:
I'm really sorry but could you change the folder name instead of the package name ?

Singular is preffered in Go

Reply:
Do you mean the name of the generated folder?

Reply:
I am happy to do this, but we must consider that this will be a breaking change for anyone who has an existing `fuego` project. The updated CLI would write new controllers in the `controller` folder, while the existing folder would be called `controllers`.

Reply:
Yes ! People will rename their folder, it's not costly

The other way around was also a breaking change

Reply:
Done!